docs/ossec_alerts.rst

```diff
@@ -475,6 +475,35 @@ email alert every morning containing text similar to: ::
 
 This is a normal alert, and informs you that the system is working as expected.
 
+Similarly, every few weeks your SecureDrop Application and Monitoring
+Servers should update the installed applications on your servers. Below is a
+sample alert, but you may see any number of these records in the logs. This
+will happen in batches so these emails might be longer than the below alert.
+You should also see them in an email named ``Daily Report: File Changes``:  ::
+
+    Received From: (app)
+    Rule: 2902 fired (level 7) -> "New (Debian Package) installed."
+    Portion of the log(s):
+
+    status installed <package name> <version>
+
+This is a normal alert, it tells you your system is up-to-date and patched.
```

Comment:
We should probably recommend a path for further investigation here, e.g. confirming entries in apt/dpkg logs, as well as linking out to the trusty package list so we can use an external source of truth to verify the authenticity of the updates reported by OSSEC.